components/collector/src/source_collectors/__init__.py

```diff
@@ -114,6 +114,7 @@
 from .pyupio_safety.security_warnings import PyupioSafetySecurityWarnings
 
 from .quality_time.metrics import QualityTimeMetrics
+from .quality_time.completeness import QualityTimeCompleteness
```

Comment:
I thought we decided to rename the metric to "missing metrics"?
```suggestion
from .quality_time.completeness import QualityTimeMissingMetrics
```